### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TakashiL**
Correct a few grammar mistake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ docker-compose up --build
 
 that's it :)
 
-You can also follow below instructions if you want it to be more customizable
+You can also follow instructions below if you want customize it.
```
Comment (Armour) [inline README.md]: add 'to' maybe

File: README.md [hunk 2]
```diff
@@ -83,17 +83,17 @@ yarn install
 yarn global add gulp
 ```
 
-If you meet permission problem when trying to install yarn globally, check [this](https://github.com/yarnpkg/yarn/issues/1060#issuecomment-268160528) out.
+If you find permission problem when trying to install yarn globally, check [this](https://github.com/yarnpkg/yarn/issues/1060#issuecomment-268160528) out.
 
 ### Setup database and session store
 
-You can either run `postgresql` and `redis` using docker images:
+You can either run `postgresql` or `redis` using docker images:
```
Comment (Armour) [inline README.md]: should still be and, below has an 'or' (or maintain it by yourself) for the 'either' above

File: README.md (hunk 1 above)
Comment (TakashiL) [reply README.md]: Added 'to'.

File: README.md (hunk 2 above)
Comment (TakashiL) [reply README.md]: Correct it, and added two bullets to make it looks more clear.

**Pull request merged**